### PR TITLE
add check mandatory parameters

### PR DESCRIPTION
**Pull request by mnecas**

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by mwperina** [comment]
ci add to whitelist

File: tasks/main.yml [hunk 1]
```diff
@@ -1,5 +1,20 @@
 ---
 - block:
+    - name: Check if mandatory parameters are correct
+      fail:
+        msg: "mandatory parameter is undefined '{{item}}' "
```
Comment (machacekondra) [inline tasks/main.yml]:
```
"mandatory parameter '{{ item }}' is undefined"
```

File: tasks/main.yml [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+      when: "{{item}} is undefined"
```
Comment (machacekondra) [inline tasks/main.yml]: Please use spaces around `{{` and `}}` on all places.

File: tasks/main.yml [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+      with_items:
+        - [data_center_name, compatibility_version ]
+    
+    - name: Check if hosts are correct
+      fail:
+        msg:"'host' does not contain mandatory parameter '{{ item[1] }}' "
```
Comment (machacekondra) [inline tasks/main.yml]: please add space after ':'

Comment (machacekondra) [inline tasks/main.yml]: "'hosts' variable does not contain mandatory parameter '{{ item[1] }}'"

File: tasks/main.yml [hunk 4]
```diff
@@ -1,5 +1,20 @@
 ---
 - block:
+    - name: Check if mandatory parameters are correct
+      fail:
+        msg: "mandatory parameter '{{ item }}' is undefined"
+      when: "{{ item }} is undefined"
```
Comment (machacekondra) [inline tasks/main.yml]: I think `{{` `}}` should be around `item` doesn't it says any warnings?

File: tasks/main.yml [hunk 5]
```diff
@@ -1,5 +1,20 @@
...
+      with_items:
+        - [data_center_name, compatibility_version ]
```
Comment (machacekondra) [inline tasks/main.yml]: please remove trailing space after `compatibility_version`

**Review by machacekondra — APPROVED** [review]

**Pull request merged**